src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Perf.Comments.cs

```diff
@@ -41,18 +39,14 @@ public partial class Utf8JsonReaderCommentsTests
         public void Setup()
         {
             _jsonPayload = Encoding.UTF8.GetBytes(s_testCases[TestCase]);
-
-            if (MultiSegment)
-            {
-                _jsonPayloadSequence = GetSequence(_jsonPayload, SegmentSize);
-            }
+            _jsonPayloadSequence = GetSequence(_jsonPayload, 100);
         }
 
         [Benchmark]
-        public void Utf8JsonReaderCommentParsing()
+        public void EmptyLoop()
```

Comment:
ConsumeAll perhaps

Reply:
Maybe. However, we don't really "consume". I would think consume to mean observing each token (either via `reader.ValueSpan` or reader.GetString/GetInt/etc.)

Reply:
for me, reading is consuming (also matches naming convention used in the product)